src/platforms/flutter/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -212,7 +212,7 @@ SentryAssetBundle(enableStructuredDataTracing: false)
 
 ### File I/O Instrumentation
 
-The Sentry-specific file I/O implementation starts a span out of the active span, bound to the scope for each file I/O operation. The SDK sets the span `operation` to `file.copy`, `file.write`, `file.delete`, `file.open`, `file.read` or `file.rename`, and `description` to `filename` (for example, `file.txt`).
+The Sentry-specific file I/O implementation starts a span out of the active transaction, bound to the scope for each file I/O operation. The SDK sets the span `operation` to `file.copy`, `file.write`, `file.delete`, `file.open`, `file.read` or `file.rename`, and `description` to `filename` (for example, `file.txt`).
 
 In addition, the span contains other useful information such as `file.size` (raw number of bytes), `file.path` (an absolute path to the file) and `file.async` (`true` if the called method returns a `Future`, or `false` if it's a `Sync` call) as part of the `data` payload.
```

Comment:
```suggestion
In addition, the span contains other useful information as part of the `data` payload, such as: `file.size`, (the raw number of bytes), `file.path`, (an absolute path to the file), and `file.async`, (which is `true` if the called method returns a `Future`, or `false` if it's a `Sync` call).
```

Reply:
This text has not changed, it was just a typo from `span ` to `transaction `.
Can we do it in a different PR?